vendor/golang.org/x/crypto/pkcs12/pkcs12.go

```diff
@@ -135,6 +142,9 @@ func convertBag(bag *safeBag, password []byte) (*pem.Block, error) {
 
 	for _, attribute := range bag.Attributes {
 		k, v, err := convertAttribute(&attribute)
+		if err == errUnknownAttributeOID {
+			continue
+		}
```

Comment:
Not sure if this affects us; looks like there already was a similar error further down